src/libcrun/linux.c

```diff
@@ -960,16 +960,17 @@ struct symlink_s
 {
   const char *path;
   const char *target;
+  bool force;
 };
 
 static struct symlink_s symlinks[] =
   {
-    {"/proc/self/fd", "fd"},
-    {"/proc/self/fd/0", "stdin"},
-    {"/proc/self/fd/1", "stdout"},
-    {"/proc/self/fd/2", "stderr"},
-    {"/proc/kcore", "core"},
-    {"pts/ptmx", "ptmx"},
+    {"/proc/self/fd", "fd", false},
+    {"/proc/self/fd/0", "stdin", false},
+    {"/proc/self/fd/1", "stdout", false},
+    {"/proc/self/fd/2", "stderr", false},
+    {"/proc/kcore", "core", false},
+    {"pts/ptmx", "ptmx", true},
     {NULL, NULL}
```

Comment:
This needs to be `{NULL, NULL, false}` to suppress the warning that causes tests to fail.

Reply:
thanks, I've pushed a new version with the change you've suggested